differs/pipDiff.go

```diff
@@ -37,27 +39,50 @@ func getPythonVersion(pathToLayer string) ([]string, error) {
 	return matches, nil
 }
 
-func (d PipDiffer) getPackages(path string) (map[string]map[string]utils.PackageInfo, error) {
-	packages := make(map[string]map[string]utils.PackageInfo)
+func getPythonPaths(vars []string) []string {
```

Comment:
My concern was that if a user changes their PYTHONPATH many different times and installs things in many different places, would we be finding all of those packages?